pages/07.Addons/02.metakube-ingress/default.en.md

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Ingress Controller'
+published: false
+date: '18-07-2019 13:00'
+taxonomy:
+    tag:
+        - metakube
+        - ingress
+        - nginx
+---
+
+With the NGINX Kubernetes Ingress controller, you get basic HTTP and TCP load balancing and SSL/TLS termination for all Kubernetes Ingress resources.
+
+You can find more information on ingresses and the nginx ingress controller and all it's configuration options at:
+
+* the [Kubernetes Ingress Documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+* the [Nginx Ingress Controller Documentation](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/)
+* the tutorial [Deploy an Application](../../03.Tutorials/16.deploy-an-application/default.en.md)
```

Comment:
This tutorial requires an ingress and cert manager, it should be at least mentioned, that both addons should be installed

Reply:
I'd say we put this information on the tutorial page.

Reply:
I added it to the tutorial, what do you think?

The cert-manager addon information would be added there as soon as we have the cert-manager addon.